app/controllers/posts_controller.rb

```diff
@@ -11,9 +11,23 @@ def show
     @post = @user.posts.find(params[:id])
   end
 
+  def create
+    @post = current_user.posts.build(post_params)
+
+    if @post.save
+      redirect_to root_path, notice: 'Post created!'
+    else
+      render :new
+    end
+  end
+
```

Comment:
- [ ] You will need a new controller method here so as to allow users to access create a new post form and that is what this miles stone is all about, kindly include a method that looks similar to this `@post = Post.new`, because currently when I try to create a new post I get an error
![Screenshot from 2023-11-25 09-42-50](https://github.com/Omwomo/eugene-mindscape/assets/66097371/07620bad-cdac-4f72-965b-9aee802a2a5e)